docs/pages/router/reference/api-routes.mdx

```diff
@@ -211,6 +211,156 @@ You can deploy the server for production using [EAS Hosting](/eas/hosting/get-st
+
+```sh .env
+EXPO_UNSTABLE_DEPLOY_SERVER=1
+# @info <a target="_blank" href="https://expo.dev/accounts/[account]/settings/access-tokens" style="text-decoration:underline;">Setup the token</a> #
+EXPO_TOKEN=xxxxxxx
+# @end #
+```
+
+</Step>
+
+<Step label="5">
+
+Set the `owner` field in the `app.json` to the owner of the project (this is usually your EAS username).
+
+```json app.json
+{
+  "expo": {
+    "owner": "bacon"
+  }
+}
+```
+
+</Step>
+
+<Step label="6">
+
+You're now ready to use automatic server deployment! Run the build command to start the process.
+
+<Terminal cmd={['$ eas build']} />
+
+You can also run this locally with:
+
+<Terminal
+  cmd={[
+    '# iOS',
+    '$ npx expo run:ios --configuration Release',
+    '# Android',
+    '$ npx expo run:android --variant release',
+  ]}
+/>
+
+</Step>
+
+Notes about automatic server deployment for native apps:
+
+- Server failures may occur during the `Bundle JavaScript` phase of EAS Build if something was not setup correctly.
+- You can manually deploy the server and set the `origin` URL before building the app if you'd like.
+- Automatic deployment can be force skipped with the environment variable `EXPO_NO_DEPLOY=1`.
+- Automatic deployment does not support `app.config.js` files yet.
+- Logs from the deployment will be written to `.expo/logs/deploy.log`.
+- Deployment will not run in `EXPO_OFFLINE` mode.
+
+### Testing the native production app locally
+
+It can often be useful to test the production build against a local dev server to ensure everything is working as expected. This can speed up the debugging process substantially.
+
+<Step label="1">
+
+First, export the production server:
+
+<Terminal cmd={['$ npx expo export']} />
+
+</Step>
+<Step label="2">
+
+Host the production server locally:
+
+<Terminal cmd={['$ npx expo serve']} />
+
+</Step>
+
+<Step label="3">
+
+Set the origin in the `app.json`'s `origin` field. Ensure no generated value is in `expo.extra.router.origin`. This should be `http://localhost:8081` (assuming `npx expo serve` is running on the default port).
+
+```json app.json
+{
+  "expo": {
+    "plugins": [
+      [
+        "expo-router",
+        {
+          "origin": "http://localhost:8081"
+        }
+      ]
+    ]
+  }
+}
+```
+
+Remember to remove this `origin` value when deploying to production!
+
+</Step>
+
+<Step label="4">
+
+Build the app in release mode on to a simulator:
+
+<Terminal cmd={['$ EXPO_NO_DEPLOY=1 npx expo run:ios --configuration Release']} />
+
+</Step>
+
+You should now see requests coming in to the local server. Use a tool like [Proxyman](https://proxyman.com/) to inspect network traffic for the simulator and gain better insight.
+
+You can experimentally change the URL and quickly rebuild for iOS using the `--unstable-rebundle` flag. This will swap out the app.json and client assets for a new ones, skipping the native rebuild.
```

Comment:
```suggestion
You can experimentally change the URL and quickly rebuild for iOS using the `--unstable-rebundle` flag. This will swap out the **app.json** and client assets for new ones, skipping the native rebuild.
```

Let's make filenames bold as per our writing style guide.